front/src/modules/simulationResult/components/TrainDetails.tsx

```diff
@@ -30,33 +31,48 @@ const getOccupancyBounds = (
 };
 
 export default function TrainDetails() {
-  const { positionValues, timePosition, consolidatedSimulation, selectedTrainId } = useSelector(
+  const { consolidatedSimulation, selectedTrainId } = useSelector(
     (state: RootState) => state.osrdsimulation
   );
 
+  const [localTimePosition, setLocalTimePosition] = useState<Date>(new Date());
```

Comment:
No need for the localTimePosition (it is not used)
```suggestion
```

Reply:
nicely spotted, done